rollup.config.js

```diff
@@ -0,0 +1,35 @@
+/* jshint multistr:true, node:true */
+
+var babel = require( "rollup-plugin-babel" );
+
+module.exports = {
+	format: "iife",
+	exports: "none",
+	plugins: [
+		babel( {
+			presets: [ "es2015-rollup" ]
+		} )
+	],
+
+	// jscs:disable disallowMultipleLineStrings
+	banner: "\
+/*!\n\
+ * QUnit @VERSION\n\
+ * https://qunitjs.com/\n\
+ *\n\
+ * Copyright jQuery Foundation and other contributors\n\
+ * Released under the MIT license\n\
+ * https://jquery.org/license\n\
+ *\n\
+ * Date: @DATE\n\
```

Comment:
Not quite, there's also [one instance in `src/core`](https://github.com/jquery/qunit/blob/b7a00ace99f60a1073a7fcf47312cc80edc556ca/src/core.js#L5), which is why I didn't already drop the concat step.